FedoraRelationships.php

```diff
@@ -799,6 +799,9 @@ public function add($predicate_uri, $predicate, $object, $type = RELS_TYPE_URI)
    *   TRUE if relationships were removed, FALSE otherwise.
    */
   public function remove($predicate_uri = NULL, $predicate = NULL, $object = NULL, $type = RELS_TYPE_URI) {
+    if (!isset($this->aboutDs->parent['RELS-INT'])) {
+      return FALSE;
+    }
     $this->initializeDatastream();
```

Comment:
@jordandukart so this here removes the auto creation of RELS-INT in case not explictly defined? I see it returns, in case it is not there, making
https://github.com/Islandora/tuque/blob/d34bb4b25d2c30f5795d47ae496d53f7600a4bca/FedoraRelationships.php#L746-L760 obsolete really in this context (still makes sense when adding, get, etc)
Your changes make sense, why remove or even try to remove if not there.